convert2rhel/systeminfo.py

```diff
@@ -33,11 +35,26 @@
 from convert2rhel.toolopts import tool_opts
 
 
+# Supported conversion path to RHEL
+RELEASE_VER_MAPPING = {
+    "CentOS Linux": {
+        "8.4": "8.4",
+        "8.3": "8.3",
+        "7.9": "7.9Server",
+    },
+    "Oracle Linux Server": {
+        "8.4": "8.4",
+        "8.3": "8.3",
+        "7.9": "7.9Server",
```

Comment:
the same

Reply:
`7.9` releasever repo contains RHEL 7.9 GA packages, whereas `7Server` contains the latest RHEL 7.9 package updates. That's what we need.

It's not the case for `6.10` and `8.x` releasevers. Since RHEL 6.10 is the latest minor version, we can safely use `6Server` and leave it like that forever. For `8.x`, we can and need to specify the specific minor version (i.e. `8.4` releasever contains the latest RHEL 8.4 package updates, not just GA versions).